cmd/cluster-network-operator/constants.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package k8sutil
+package main
```

Comment:
can you just move whichever constants we care about into the other file?

likewise, drop any bits from the other files that we aren't using

Reply:
Which other file, you mean any constant within cmd/cluster-network-operator/* onto constants.go ?

Reply:
whichever other file(s) were using the constants from here. (I was thinking when I wrote the comment that you were only copying over leader.go, so that's what I meant...)

maybe it makes sense to keep it as a separate file but in that case maybe the new files should all move to `pkg/util/operatorsdk` or something like that